component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/MetricSchema.java

```diff
@@ -24,5 +24,8 @@
 public enum MetricSchema {
     RESPONSE,
     ERROR,
-    CUSTOM
+    LATENCY,
+    PAYLOAD,
+    CUSTOM1,
```

Comment:
What's the reason for having as CUSTOM1 and CUSTOM2?

Reply:
I just added to try out the schema adding capability. since we are not shipping that will remove the additional ones